src/module.c

```diff
@@ -1151,6 +1151,7 @@ int64_t commandFlagsFromString(char *s) {
         else if (!strcasecmp(t,"no-cluster")) flags |= CMD_MODULE_NO_CLUSTER;
         else if (!strcasecmp(t,"no-mandatory-keys")) flags |= CMD_NO_MANDATORY_KEYS;
         else if (!strcasecmp(t,"allow-busy")) flags |= CMD_ALLOW_BUSY;
+        else if (!strcasecmp(t, "internal")) flags |= (CMD_INTERNAL|CMD_NOSCRIPT); // We disallow internal commands in scripts.
```

Comment:
Wonder why we did not do the same trick with `no-monitor` and `no-slowlog`. I believe we should at least be consistent. I you ask me, it is better to check for the internal flag where we check the `CMD_NOSCRIPT` other then explicitly add flags to the command. It should not be a lot of places.

Reply:
We decided to start off with statistics being reported for internal commands, e.g., monitor and slow-log.
Yes it's not a lot of places, I can modify the conditions explicitly, although I think this is cleaner.

Reply:
I think that it should not be hard to support slow-log and monitor.

Reply:
What do you mean support? Remove the internal commands from there?

Reply:
Yes